### PR TITLE
catty-board assignment submission

### PR DESCRIPTION
**Pull request by lmallikarjuna**

### CODE REVIEW

File: Week-1/catty-board/index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+              <p>2016.Charts.100 songs</p>
+            </section>
+            <section class="info-card-buttons">
+              <button><i class="la la-play"></i>Play All</button>
+              <button><i class="la la-plus"></i>Add to</button>
+              <button><i class="la la-ellipsis-h"></i>More</button>
+            </section>
+          </div>
+        </div>
+        <!-- <div class="list-container">
```
Comment (sumitbusa) [inline Week-1/catty-board/index.html]: Remove the commented code

File: Week-1/catty-board/index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+      </nav>
+      <main>
+        <div class="info-card">
+          <img
+            src="https://vif1g.csb.app/src/assets/cattyboard.jpg"
+            alt="catty-board"
+            class="top-image"
+          />
+          <div class="info-card-info">
+            <section>
```
Comment (sumitbusa) [inline Week-1/catty-board/index.html]: use of the session tag is for stand-alone module or page.

File: Week-1/catty-board/index.html [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+          </div>
+          <div class="card card-even">
+            <p class="left-one">1.One Dance</p>
+            <p class="left-two">ASDFASDFASDF</p>
+            <p class="right-one">2:54</p>
+            <button class="right-two">CATTY CLOUD SYNC</button>
+          </div>
+        </div> -->
+        <table>
+          <tr></tr>
```
Comment (sumitbusa) [inline Week-1/catty-board/index.html]: why blank <tr> tag? try to avoid blank tag.

File: Week-1/catty-board/style.css [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+  text-align: left;
+  padding: 8px;
+}
+
+tr:nth-child(even) {
+  background-color: #dddddd;
+}
+
+tr > td:nth-child(4) {
+  color: blue;
```
Comment (sumitbusa) [inline Week-1/catty-board/style.css]: use color code of rgb value

**Review by sumitbusa — CHANGES_REQUESTED** [review]
Have a look into comment once

File: Week-1/catty-board/index.html (hunk 1 above)
Comment (lmallikarjuna) [reply Week-1/catty-board/index.html]: Resolved

File: Week-1/catty-board/index.html (hunk 2 above)
Comment (lmallikarjuna) [reply Week-1/catty-board/index.html]: Resolved

File: Week-1/catty-board/index.html (hunk 3 above)
Comment (lmallikarjuna) [reply Week-1/catty-board/index.html]: Resolved

File: Week-1/catty-board/style.css (hunk 4 above)
Comment (lmallikarjuna) [reply Week-1/catty-board/style.css]: Resolved

**Pull request merged**